Yona/Yona/Info.plist

```diff
@@ -75,6 +75,7 @@
 	<key>UIRequiredDeviceCapabilities</key>
 	<array>
 		<string>armv7</string>
+		<string>gamekit</string>
```

Comment:
Why is this capability required?

Reply:
We don't need it for our application. I will remove this.  We can add it later if we need it.